src/main/java/org/gridsuite/explore/server/services/DirectoryService.java

```diff
@@ -241,6 +241,10 @@ private IDirectoryElementsService getGenericService(String type) {
         return iDirectoryElementsService;
     }
 
+    public List<Object> getCompositeModificationContent(UUID compositeModificationId) {
+        return ((NetworkModificationService) getGenericService(MODIFICATION)).getCompositeModificationContent(compositeModificationId);
+    }
```

Comment:
why this function in directoryService ? it is not about cases 
it should be in exploreservice

Reply:
True. I moved it.